packages/owl-bot/cloud-build/scan-googleapis-gen-and-create-pull-requests.yaml

```diff
@@ -33,6 +33,28 @@ steps:
       - --global
       - user.name
       - 'Owl Bot'
+  # TODO: remove when we find a better resolution.
+  # See: https://github.com/orgs/community/discussions/55820
+  - name: 'gcr.io/cloud-builders/git'
```

Comment:
I don't think this method is going to work. GCB only shares the /workspace directory between steps (or possibly other folders if you explicitly share). `git config --global` commands puts config in your `~/.git/config`.

Reply:
I believe the git push happens in the CLI command not in the docker container:

https://github.com/googleapis/repo-automation-bots/blob/main/packages/owl-bot/src/copy-code.ts#L817